pyproject.toml

```diff
@@ -10,39 +10,34 @@ classifiers = [
   "Intended Audience :: Developers",
   "License :: OSI Approved :: MIT License",
   "Programming Language :: Python :: 3",
-  "Programming Language :: Python :: 3.8",
-  "Programming Language :: Python :: 3.9",
-  "Programming Language :: Python :: 3.10",
-  "Programming Language :: Python :: 3.11",
-  "Programming Language :: Python :: 3.12",
```

Comment:
Aren't these used by PyPI to show supported Python versions?

Reply:
This is indeed a great question!

The value of `project.requires-python` defines what is actually supported for installers.  The classifiers are only used to display/search on PyPI. I removed those to avoid the repetitive update of both fields since one of them, which is actually enforced by the installer, is already defining this information.

If users of docstrfmt use that functionality on PyPI, we should definitely recover it!

Reply:
In this case, shall I also add 3.13?

Reply:
Does libCST support Python 3.13?

Reply:
It does say so: https://github.com/Instagram/LibCST/blob/main/pyproject.toml#L20

Reply:
Conda-forge builds also support 3.13: https://github.com/conda-forge/libcst-feedstock

Reply:
Then I'm good with that!

Reply:
Done at the last commit.